storage/bootstrap.go

```diff
@@ -193,7 +206,7 @@ func (m *bootstrapManager) bootstrap() error {
 	// all the data we bootstrapped.
 	rateLimitOpts := m.fsManager.RateLimitOptions()
 	m.fsManager.SetRateLimitOptions(rateLimitOpts.SetLimitEnabled(false))
-	m.fsManager.Run(m.nowFn(), false)
+	m.fsManager.Run(m.nowFn(), false, true)
```

Comment:
When we get here, the file system manager is disabled, but we do want to flush the data out to disk, which is why the force flag is in place.